Code/Editor/Plugins/FFMPEGPlugin/main.cpp

```diff
@@ -24,11 +24,9 @@ PLUGIN_API IPlugin* CreatePluginInstance(PLUGIN_INIT_PARAM* pInitParam)
     // Make sure the ffmpeg command can be executed before registering the command
     if (!CFFMPEGPlugin::RuntimeTest())
     {
-        AZStd::string msg =
-            "FFMPEG plugin: Failed to execute FFmepg. Please run Setup Assistant, "
+        GetIEditor()->GetSystem()->GetILog()->Log("FFMPEG plugin: Failed to execute FFmepg. Please run Setup Assistant, "
             "go to the 'Optional software' section of the 'Install software' tab, "
-            "and make sure the FFmpeg executable is correctly configured.";
-        GetIEditor()->GetSystem()->GetILog()->Log(msg.c_str());
+            "and make sure the FFmpeg executable is correctly configured.");
```

Comment:
Uhhh we don't have Setup Assistant anymore :)

Reply:
will update this comment to "FFMPEG plugin: Failed to execute FFmepg. Please install FFmepg."